aiden_app/agents/agent.py

```diff
@@ -1,58 +1,101 @@
 import json
+import os
 from typing import Any, Dict, List, Tuple
 from abc import ABC, abstractmethod
 
 from ..tools.cv_editor_tool import CVEditorTool
 from ..tools.indeed_scraper_tool import IndeedScraperTool
 from ..tools.tool_aggregator import ToolAggregator
-from .prompts import START_CHAT_PROMPT, SYSTEM_PROMPT
+from ..tools.utils.cv_editor import CVEditor
+from .prompts import START_CHAT_PROMPT, SYSTEM_PROMPT, PROFILE_CREATION_SYSTEM_PROMPT
 
 
 class Agent(ABC):
-    def __init__(self, profile: Dict[str, Any], message_class: Any):
-        self.tool_aggregator = ToolAggregator(
-            [
-                CVEditorTool(first_name=profile["first_name"], last_name=profile["last_name"]),
-                IndeedScraperTool(),
-            ]
-        )
-        start_messages = [
-            message_class(role="system", content=SYSTEM_PROMPT),
-            message_class(role="system", content=json.dumps(profile)),
-            message_class(role="assistant", content=START_CHAT_PROMPT),
-        ]
+    def __init__(self, message_class: Any):
+        self.tool_aggregator = ToolAggregator([])
         self.message_class = message_class
-        self.messages: List[Dict[str, Any]] = start_messages
+        self.messages: List[Dict[str, Any]] = []
+        self.profile: Dict[str, str] = {}
```

Comment:
we can use directly `list` and `dict` with our python version but that"s not very important
```suggestion
        self.messages: list[dict[str, Any]] = []
        self.profile: dict[str, str] = {}
```

Reply:
I agree, I like it more, it's just copilot generating some outdated typing